src/index.js

```diff
@@ -325,14 +325,20 @@ class MiniCssExtractPlugin {
               source,
               '',
               `// ${pluginName} CSS loading`,
+              `var fetchRTL = !!window['${this.options.globalRTLFlag}'];`,
```

Comment:
Store, in run-time, if RTL stylesheets should be fetched. Checks `window[globalRTLFlag]` provided from passed options.

Reply:
I suppose it just made more sense for the authors to take this approach. A separate JS might have worked, but I guess since there are multiple places that code is _injected_ it was may have been easier to create this inline.